content/md/products-super-power/enrich-your-reference-entity-records.md

```diff
@@ -73,6 +78,23 @@ You can also filter on completeness in order to see the records that are complet
 To know how to define the required attributes for the completeness, please take a look at this [article](/articles/manage-reference-entities.html)
 :::
 
+## Filter on options and reference entity links (_since 3.1_)
+
+_Since 3.1_, we added the ability to filter on attributes types single/multiple options or reference entity single/multiple links in the records grid. It allows an exact search to find the records with a specific value (option value or link to a record).
+
+For examples:
```

Comment:
Je pense qu'il n'y a pas de "s" quand on met "for" avant.

Reply:
you can use "for instance", or "e.g."